### ISSUE TITLE
Discuss what we want to do about Licensing

### ISSUE THREAD

**Issue by Arsh25** [FoundYourStuff/core#46]
Making this an issue so we can track it and document anything we feel is important.

**Comment by Jlrine2** [comment]
I'm partial to BSD-3 for permissive, but we might now want to allow commercial use?

**Comment by Camology** [comment]
Yeah the idea was a free product that we can get hosting funds through ads and maybe some sort of premium membership tier/ tag system if I recall correctly. GNU seems fine to me, if people want to use it they can but would have to also use the same license if I recall correctly even if it's a commercial work and make their code open source.

**Comment by Arsh25** [comment]
The issue is do we want to allow for derivative works or not and how we want to treat them.

**Comment by Camology** [comment]
My only concern really about a derivative work would be if someone wanted to use this framework to do something nefarious like a human trafficking network? But that is already illegal so a license doesn't really matter to an illegal organization and they could already use our app for that if they really wanted to unfortunately. Don't think derivatives are that concerning at least to me at this moment
